packages/dd-trace/src/appsec/reporter.js

```diff
@@ -13,54 +14,52 @@ const {
 // default limiter, configurable with setRateLimit()
 let limiter = new Limiter(100)
 
-// TODO: use precomputed maps instead
-const REQUEST_HEADERS_PASSLIST = [
-  'accept',
-  'accept-encoding',
-  'accept-language',
+const contentHeaderList = [
   'content-encoding',
   'content-language',
   'content-length',
-  'content-type',
-  'forwarded',
-  'forwarded-for',
+  'content-type'
+]
+
+const REQUEST_HEADERS_PASSMAP = mapHeaderAndTags([
```

Comment:
you can call them just `REQUEST_HEADERS_MAP` hahaha, or just PASSLIST if you want, but passmap is just goofy